micromamba.el

```diff
@@ -194,6 +277,31 @@ The parameters value is an alist as defined by
     (setenv (car var) (cdr var)))
   (setq eshell-path-env (getenv "PATH")))
 
+;; "public" functions
+
+(defun micromamba-env-default-location ()
+  "Default location of the conda environments -- under the Anaconda installation."
+  (let ((candidates (alist-get 'envs_dirs (micromamba--get-config))))
+    (f-full (aref candidates 0))))
+
+
+(defun micromamba-env-name-to-dir (name)
+  "Translate NAME to the directory where the environment is located."
+  (if (and (string= name "base")
+           (micromamba--env-dir-is-valid micromamba-home))
+      (file-name-as-directory (expand-file-name micromamba-home))
+    (let* ((default-location (file-name-as-directory (micromamba-env-default-location)))
+           (initial-possibilities (list name (concat default-location name)))
+           (possibilities (if (boundp 'venv-location)
+                              (if (stringp venv-location)
+                                  (cons venv-location initial-possibilities)
+                                (nconc venv-location initial-possibilities))
+                            initial-possibilities))
+           (matches (-filter 'micromamba--env-dir-is-valid possibilities)))
+      (if (> (length matches) 0)
+          (file-name-as-directory (expand-file-name (car matches)))
+        (error "No such conda environment: %s" name)))))
```

Comment:
I think this should be replaced by parsing `micromamba env list --json`.

`conda.el` does this because `conda` is written in Python and as such too slow to call. `micromamba` is much, much faster.

This will allow removing a whole bunch of stuff.